.gitmodules

```diff
@@ -3,7 +3,7 @@
 	url = git@github.com:HumanCompatibleAI/gogui.git
```

Comment:
```suggestion
	url = git@github.com:AlignmentResearch/gogui.git
```

Reply:
Will wait on these repos to be migrated and then ammend this.